src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -5276,7 +5276,7 @@ void wm_vuldet_create_and_send_message(vu_task_ctx_t* task, task_status TASK_STA
                 break;
         }
         if (message) {
-            linked_queue_push_ex(vu_messages_queue, (void*) message);
+            queue_push_ex_block(vu_messages_queue, (void*) message);
```

Comment:
If the queue is full then vulnerability detector will stop working. I think this situation should be properly warned to the user.

Reply:
Added a warning message

Reply:
Great! 
I'd just modify the `Working thread will stop executing tasks` part for something more clear for the user.
We should say that `vulnerability detector` will be blocked, because the `working thread` is an internal name.